pkg/controllers/deprovisioning/events.go

```diff
@@ -56,8 +56,8 @@ func WaitingOnDeletion(node *v1.Node) events.Event {
 	return events.Event{
 		InvolvedObject: node,
 		Type:           v1.EventTypeNormal,
-		Reason:         "ConsolidateWaiting",
-		Message:        "Waiting on deletion to continue consolidation",
+		Reason:         "DeprovisioningWaiting",
```

Comment:
Looks like we have this event twice?

Reply:
This was from a previous change. Not changing anything right now.

Although there are two events that have the same reason, they have different messages, so it's technically a different event.

Reply:
Changed it to be DeprovisioningWaitingReadiness and DeprovisioningWaitingDeletion